### PR TITLE
feat: limit custom feed view from date

### PR DESCRIPTION
**Pull request by capJavert**
AS-874

also not removing default custom feed if not plus

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]

File: .infra/Pulumi.adhoc.yaml [hunk 1]
```diff
@@ -84,6 +84,7 @@ config:
     typeormUsername: postgres
     tz: UTC
     urlPrefix: http://api.local.com
+  api:image: api-image:tilt-3c540e5fe45b2a14
```
Comment (sshanzel) [inline .infra/Pulumi.adhoc.yaml:87]: Accidentally pushed I think.

File: src/routes/boot.ts [hunk 2]
```diff
@@ -510,6 +510,7 @@ const loggedInBoot = async ({
         language: user.language || undefined,
         image: mapCloudinaryUrl(user.image),
         cover: mapCloudinaryUrl(user.cover),
+        defaultFeedId: isPlus ? user.defaultFeedId : null,
```
Comment (sshanzel) [inline src/routes/boot.ts:513]: Should we return the my-feed id (userId) or basically wouldn't really matter?

**Review by sshanzel — APPROVED** [review]
Just the accidental push, other than that it looks amazing 🚀

File: src/routes/boot.ts (hunk 2 above)
Comment (capJavert) [reply src/routes/boot.ts:513]: null is default when you don't have anything set, frontend expects it as well

File: .infra/Pulumi.adhoc.yaml (hunk 1 above)
Comment (capJavert) [reply .infra/Pulumi.adhoc.yaml]: yep, good catch, removed!

**Pull request merged**